Gemfile.base

```diff
@@ -37,7 +37,7 @@ gem 'i18n'
 gem 'multi_fetch_fragments', github: 'mikz/multi_fetch_fragments'
 
 # Apisonator client
-gem 'pisoni', '~> 1.24'
+gem 'pisoni', '1.25.0'
```

Comment:
This change is already in master

Reply:
@hallelujah Good for the loose versioning but it is not in master 😕 
https://github.com/3scale/porta/blob/master/Gemfile.base#L40
https://github.com/3scale/porta/blob/ed85cc6910815dc28df425f8e7da5145c88a1878/gemfiles/prod/Gemfile.lock#L500

Reply:
> We should use loose versioning instead of exact version.

OK 👍 

> This change is already in master

In the `lock` yes but imo this should be here `~> 1.25`
Because we know that we 1.24, it wouldn't work.